web/src/lib/components/nostr/Chat.svelte

```diff
@@ -28,7 +28,10 @@
     const nostrQueriesBatchSize = 100;
     const nostrOrderMessagesDelay = 1500;
     const nostrBackgroundJobsDelay = 3000;
-    const nostrMediaCacheEnabled = true;
+    const nostrMediaCacheEnabled = false;
+
+    let timerOrderMessages = null;
```

Comment:
The order in English is different... attribute + noun. So this would be "orderMessagesTimer"...

Reply:
You're right. I just wanted that it had the type at the beggining, but it appears weird. I'll change it.